lms/djangoapps/dashboard/tests/test_sysadmin.py

```diff
@@ -300,6 +300,10 @@ def test_gitlog_pagination_out_of_range_invalid(self):
 
         mongoengine.connect(TEST_MONGODB_LOG['db'])
 
+        # Clean import logs to ensure courses fall within 2 page limit
+        import_logs = CourseImportLog.objects.all()
```

Comment:
I do see your comment "Had to upgrade django-babel-underscore and i18n-tools dependencies to prevent i18n strings in underscore templates from compiling to dict with callback arg" but I'm unclear as to its effect in the non-CircleCI realm. 
I'm trying to avoid changes coming in through a PR entitled "Fixes successfully build on circleci" that are unrelated to building on CircleCI because that will make it more confusing for other devs down the road.